app/controllers/main/routes.py

```diff
@@ -12,30 +13,27 @@ def home():
     print(f"{g.current_user.username}: {g.current_user.firstName} {g.current_user.lastName}")
     return render_template('main/home.html', title="Welcome to CELTS!")
 
-@main_bp.route('/volunteerIndicateInterest', methods = ['GET'])
-def volunteerIndicateInterest():
-    programs = Program.select()
-    interests = Interest.select().where(Interest.user == g.current_user)
-    interests_ids = [interest.program for interest in interests]
-    return render_template('volunteerIndicateInterest.html',
-                           title="Volunteer Interest",
-                           user = g.current_user,
-                           programs = programs,
-                           interests = interests,
-                           interests_ids = interests_ids)
-
-
 @main_bp.route('/profile/<username>', methods = ['GET'])
 def profilePage(username):
-
-    if username == g.current_user.username or g.current_user.isCeltsAdmin or g.current_user.isCeltsStudentStaff:
-        return f'<h1>Profile page for {username}</h1>'
-
-    abort(403)
+    if not g.current_user.isCeltsAdmin or g.current_user.isCeltsStudentStaff or g.current_user.username == username:
+        print(g.current_user.username)
```

Comment:
Print more than just the username if you are wanting to print something. Otherwise remove it.